chpl/chpl-auth/src/main/java/gov/healthit/chpl/auth/entity/UserEntity.java

```diff
@@ -73,6 +75,11 @@
     @ManyToOne(optional = false, fetch = FetchType.EAGER)
     @JoinColumn(name = "contact_id", unique = true, nullable = false)
     private UserContactEntity contact;
+    
+    @ManyToMany
+    @JoinTable(name = "user_certification_body_map",
+            joinColumns = { @JoinColumn(name = "user_id") })
```

Comment:
I don't think this file should be part of this PR

Reply:
Addressed